<ui version="4.0" >
 <class>anaPencere</class>
 <widget class="QMainWindow" name="anaPencere" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>qPod</string>
  </property>
  <property name="windowIcon" >
   <iconset>resimler/qpod.png</iconset>
  </property>
  <property name="styleSheet" >
   <string>background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QWidget" native="1" name="widget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QPushButton" name="baglanButton" >
     <property name="geometry" >
      <rect>
       <x>360</x>
       <y>40</y>
       <width>38</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>resimler/baglan.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="labelBaglan" >
     <property name="geometry" >
      <rect>
       <x>210</x>
       <y>48</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet" >
      <string>background:none;</string>
     </property>
     <property name="text" >
      <string>Bağlan</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ayarButton" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>360</x>
       <y>3</y>
       <width>38</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>resimler/secenekler.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="labelAyarlar" >
     <property name="geometry" >
      <rect>
       <x>239</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet" >
      <string>background:none;</string>
     </property>
     <property name="text" >
      <string>Ayarlar</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>2</x>
      <y>80</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>background-color:white;</string>
    </property>
    <property name="tabPosition" >
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <property name="elideMode" >
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_4" >
     <attribute name="title" >
      <string>Yüklenecek Şarkılar</string>
     </attribute>
     <widget class="QListWidget" name="yuklenecekListe" >
      <property name="geometry" >
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>366</width>
        <height>221</height>
       </rect>
      </property>
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
      <property name="styleSheet" >
       <string>font-size:10px;</string>
      </property>
      <property name="dragEnabled" >
       <bool>true</bool>
      </property>
      <property name="dragDropMode" >
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="sarkilariYukleButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>221</x>
        <y>220</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text" >
       <string>Şarkıları iPod'a Yükle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sarkiSecButton" >
      <property name="geometry" >
       <rect>
        <x>109</x>
        <y>220</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text" >
       <string>Şarkı Seç</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="sarkiProgress" >
      <property name="geometry" >
       <rect>
        <x>8</x>
        <y>222</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabIpod" >
     <attribute name="title" >
      <string>iPod</string>
     </attribute>
     <widget class="QTableWidget" name="ipodListe" >
      <property name="geometry" >
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>366</width>
        <height>221</height>
       </rect>
      </property>
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet" >
       <string>font-size:9px;</string>
      </property>
      <property name="verticalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="autoScroll" >
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors" >
       <bool>false</bool>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="sarkilariSilButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>151</x>
        <y>220</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Seçili Şarkıları Sil</string>
      </property>
      <property name="flat" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="silProgressBar" >
      <property name="geometry" >
       <rect>
        <x>8</x>
        <y>222</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="yukariAlButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>103</x>
        <y>220</y>
        <width>22</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset>resimler/yukari.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="asagiAlButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>127</x>
        <y>220</y>
        <width>22</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset>resimler/asagi.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAyarlar" >
     <attribute name="title" >
      <string>Ayarlar</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidgetAyarlar" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>0</y>
        <width>361</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet" >
       <string>background-color:white;</string>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Aygıtlar</string>
       </attribute>
       <widget class="QLabel" name="label_9" >
        <property name="geometry" >
         <rect>
          <x>24</x>
          <y>14</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Aygıt</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_10" >
        <property name="geometry" >
         <rect>
          <x>14</x>
          <y>45</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Bağlama Noktası</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineAygit" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>10</y>
          <width>211</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineBaglamaNoktasi" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>40</y>
          <width>211</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="aygitlarUygulaButton" >
        <property name="geometry" >
         <rect>
          <x>239</x>
          <y>155</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text" >
         <string>Uygula</string>
        </property>
       </widget>
       <widget class="QPushButton" name="aygitlarOntanimliButton" >
        <property name="geometry" >
         <rect>
          <x>120</x>
          <y>155</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text" >
         <string>Öntanımlı</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>gnupod</string>
       </attribute>
       <widget class="QPushButton" name="gnupodAyarlarUygula" >
        <property name="geometry" >
         <rect>
          <x>239</x>
          <y>155</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text" >
         <string>Uygula</string>
        </property>
       </widget>
       <widget class="QPushButton" name="gnupodAyarlarOntanimliButton" >
        <property name="geometry" >
         <rect>
          <x>120</x>
          <y>155</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text" >
         <string>Öntanımlı</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineGnupod_INIT" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>7</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4" >
        <property name="geometry" >
         <rect>
          <x>47</x>
          <y>12</y>
          <width>76</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>tunes2pod</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5" >
        <property name="geometry" >
         <rect>
          <x>14</x>
          <y>41</y>
          <width>109</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>gnupod_addsong</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6" >
        <property name="geometry" >
         <rect>
          <x>26</x>
          <y>71</y>
          <width>97</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>gnupod_search</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7" >
        <property name="geometry" >
         <rect>
          <x>33</x>
          <y>102</y>
          <width>91</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>gnupod_check</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8" >
        <property name="geometry" >
         <rect>
          <x>70</x>
          <y>132</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>mktunes</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineGnupod_addsong" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>37</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineGnupod_search" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>67</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineGnupod_check" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>97</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineGnupod_mktunes" >
        <property name="geometry" >
         <rect>
          <x>130</x>
          <y>127</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <attribute name="title" >
        <string>Dil</string>
       </attribute>
       <widget class="QComboBox" name="dilComboBox" >
        <property name="geometry" >
         <rect>
          <x>111</x>
          <y>57</y>
          <width>141</width>
          <height>26</height>
         </rect>
        </property>
        <item>
         <property name="text" >
          <string>English</string>
         </property>
         <property name="icon" >
          <iconset>resimler/enBayrak.png</iconset>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Türkçe</string>
         </property>
         <property name="icon" >
          <iconset>resimler/trBayrak.png</iconset>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="dilUygulaButton" >
        <property name="geometry" >
         <rect>
          <x>140</x>
          <y>100</y>
          <width>80</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text" >
         <string>Uygula</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="ayarlariKapatButton" >
      <property name="geometry" >
       <rect>
        <x>229</x>
        <y>217</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text" >
       <string>Ayarları Kapat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ayarlarYardimButton" >
      <property name="geometry" >
       <rect>
        <x>1</x>
        <y>215</y>
        <width>38</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset>resimler/yardim.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
